ports/ada-url/vcpkg.json

```
@@ -0,0 +1,22 @@
+{
+  "name": "ada-url",
+  "version": "2.4.2",
+  "description": "WHATWG-compliant and fast URL parser written in modern C++",
+  "homepage": "https://ada-url.com/",
+  "license": "MIT",
+  "dependencies": [
+    {
+      "name": "vcpkg-cmake",
+      "host": true
+    },
+    {
+      "name": "vcpkg-cmake-config",
+      "host": true
+    }
+  ],
+  "features": {
+    "tools": {
+      "description": "Build CLI tools (adaparse)"
```

```suggestion
      "description": "Build CLI tools (adaparse)",
      "supports": "!uwp"
```
ada-url[core,tools]:x64-uwp still failed with the error C2039: 'output_file': is not a member of 'fmt'.